src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -4,53 +4,147 @@ You can take specific actions when users sign-in or sign-out by subscribing auth
 
```

Comment:
## Subscribing **to** Events
You can take specific actions when users sign-in or sign-out by subscribing **to** authentication events in your app. Please see our [Hub Module Developer Guide](/lib/utilities/hub) for more information.